deploy/contents/install/app/install.sh

```diff
@@ -1369,5 +1372,19 @@ if is_service_requested cp-storage-lifecycle-service; then
     echo
 fi
 
+if is_deployment_type_requested aws_eks; then
+
+   if [ $CP_EKS_CSI_DRIVER_TYPE = "efs"] && [ -n $CP_EKS_EFS_CSI_EXECUTION_ROLE]; then
```

Comment:
these two IFs should be rewritten in one if-else block
I believe instead of `CP_EKS_EFS_CSI_EXECUTION_ROLE` we have `CP_EKS_CSI_EXECUTION_ROLE` and it is common and can be used for both deployments `efs` and `fsx`